cms/cms_toolbars.py

```diff
@@ -472,10 +484,11 @@ def add_page_menu(self):
             refresh = self.toolbar.REFRESH_PAGE
 
             # menu for current page
-            current_page_menu = self.toolbar.get_or_create_menu(PAGE_MENU_IDENTIFIER, _('Page'), position=1)
+            current_page_menu = self.toolbar.get_or_create_menu(
+                PAGE_MENU_IDENTIFIER, _('Page'), position=1, disabled=self.in_apphook() and not self.in_apphook_root())
```

Comment:
please add comments as to why disabled=(conditions) is needed.

ie. "hiding page menu if page is apphook page view"